CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+Unreleased changes
+------------------
+* Add --retry flag. Specifies number of retry attempts for non-successful API responses
```

Comment:
Can we be explicit here and say `Add --retry flag. Specifies the number of retry attempts for non-successful API responses (500, 503, and timeouts only)`